tests/sandbox/sandbox.sh

```diff
@@ -11,19 +11,30 @@ VERSION_FILE="${PATH_HERE}"/../share/VERSION
 GOV_SCRIPT="${PATH_HERE}"/sandbox_gov.lua
 
 is_package_specified=false
-for item in "$@" ; do
-    if [ "$item" == "-p" ] || [ "$item" == "--package" ]; then
-        is_package_specified=true
-    fi
-done
+is_js_bundle_specified=false
 
 extra_args=("$@")
```

Comment:
So we don't hit this issue again every time we try to modify these args, I think we want all the args produced by this script to be overridable (ie - to happen _before_ user-supplied args). So the final command should be
```
exec python
  --binary-dir ...
  ...
  "${extra_args[@]}"
  "$@"
```
rather than combining the user-supplied args within `extra_args`.